PropertyBinder/Engine/BindingExecutor.cs

```diff
@@ -100,7 +111,19 @@ protected override void ExecuteInternal(BindingMap map, int[] bindings)
                     {
                         ref BindingReference binding = ref _scheduledBindings.DequeueRef();
                         binding.UnSchedule();
-                        binding.Execute();
+                        try
+                        {
+                            binding.Execute();
+                        }
+                        catch (Exception e)
+                        {
+                            var exceptionEventArgs = new ExceptionEventArgs(e);
```

Comment:
Another thought: this event would hardly allow to identify the failed binding. Please consider adding a binding description to ExceptionEventArgs so it can be logged appropriately.
You can get the description (normally containing the full expression, at least for property bindings) via binding.DebugContext?.Description

Reply:
agreed, added